packages/react-sdk/src/components/SlideOverviewBar/withContextMenu.tsx

```diff
@@ -98,6 +99,12 @@ export const withContextMenu = <P extends object>(
       handleClose();
     }, [handleClose, slideId, whiteboardInstance]);
 
+    const handleInsertSlide = useCallback(() => {
+      const slideId = whiteboardInstance.addSlide(slideIndex - 1);
```

Comment:
This should be `slideIndex + 1` - current implementation (from video capture) seemed to work because you opened the context menu while on top of the first slide but actually it was the second slide that was selected.

I would also change `slideId` to `newSlideId` so it doesn't get confused with the component property with the same name.

Reply:
> This should be slideIndex + 1 - current implementation (from video capture) seemed to work because you opened the context menu while on top of the first slide but actually it was the second slide that was selected.

I don't get it. It should add the new slide after the slide, whose context menu I have clicked?